e2etests/orchestration/common/common.go

```diff
@@ -203,6 +203,11 @@ func (h *DockerHelper) ExecADBShellCommand(id, adbBin, serial string, cmd []stri
 	return h.exec(id, append([]string{adbBin, "-s", serial, "shell"}, cmd...))
 }
 
+func (h *DockerHelper) VerifySymlink(id, filename string) error {
+	_, err := h.exec(id, []string{"readlink", filename})
```

Comment:
why not [os.Readlink](https://pkg.go.dev/os#Readlink)?

Reply:
The `readlink` command is executed within the container, in order to use `os.Readlink` we need to make the file accessible to the host by binding a mount volume when running the container.